packages/gafl-webapp-service/src/handlers/__tests__/attribution-handler.spec.js

```diff
@@ -27,20 +27,6 @@ describe('The attribution handler', () => {
     })
   })
 
-  it.each([
```

Comment:
Not sure why these tests have been removed?

Reply:
Functionality replaced by disinfect.

Reply:
But there are no tests to show that Disinfect is performing the required functionality

Reply:
It does work. Why should I have to build tests for 3rd party libraries, its not my code - it is part of hapi.

Reply:
I guess the purpose would be to check that the integration is working as expected.  Was just a little surprised to see existing tests removed.

Reply:
Phil has the handlers directly mocked so his tests could not be adapted. I have now added tests to make sure the sanitizer works for all vectors